libgdf/src/bench/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+#=============================================================================
```

Comment:
I think adding Google Bench needs to be done in a separate PR. This will also need to be made to fit @mt-jones new Cmake config.